src/main/java/org/folio/circulation/resources/DeclareLostResource.java

```diff
@@ -0,0 +1,69 @@
+package org.folio.circulation.resources;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.circulation.support.Result.failed;
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.singleValidationError;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.LoanRepository;
+import org.folio.circulation.domain.representations.DeclareItemLostRequest;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.NoContentResult;
+import org.folio.circulation.support.Result;
+import org.folio.circulation.support.http.server.WebContext;
+
+public class DeclareLostResource extends Resource {
+
```

Comment:
Personally I’d remove this blank line, as I don’t think it aids readability

Reply:
Sure.